saskatoon/harvest/forms.py

```diff
@@ -470,6 +470,14 @@ class Meta:
         required=False
     )
 
+    def __init__(self, *args, **kwargs):
+        super(PublicPropertyForm, self).__init__(*args, **kwargs)
```

Comment:
```suggestion
        super().__init__(*args, **kwargs)
```

Reply:
@tristanlatr thanks a lot for your suggestion, I have updated it. Didn't know what I used was originally Python 2 convention and we can use `super` without arguments in Python 3! Thanks!